Docker.md

```diff
@@ -0,0 +1,24 @@
+# Docker Advanced
+
+To facilitate the devoloping, using the text editor,version control and other tools already installed on your computers,
+it is possible to share files from the host into the container:
+
+```bash
+>  docker run -ti -v $(pwd):/home/porepy/shared  pmgbergen/porepy:py27
```

Comment:
why you don;t suggest this version with the display? or at least give a comment :)

docker run -ti --rm -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix  -v $(pwd):/home/porepy/shared  docker.io/pmgbergen/porepy:latest